00-preface.Rmd

```diff
@@ -42,20 +42,20 @@ knitr::include_graphics("images/flowcharts/flowchart/flowchart.002.png")
+You'll first get started with data in Chapter \@ref(getting-started) where you'll learn about the difference between R and RStudio, start coding in R, install and load your first R packages, and explore your first dataset: all domestic departure `flights` from a New York City airport in 2013. Then you'll cover the following three portions of this book (Parts 2 and 4 are combined into a single portion):
 
 1. Data science with `tidyverse`. You'll assemble your data science toolbox using `tidyverse` packages. In particular you'll
     + Ch.\@ref(viz): Visualize data using the `ggplot2` package.
     + Ch.\@ref(wrangling): Wrangle data using the `dplyr` package.
     + Ch.\@ref(tidy): Learn about the concept of "tidy" data as a standardized data frame input and output format for all packages in the `tidyverse`. Furthermore, you'll learn how to import spreadsheet files into R using the `readr` package.
-1. Data modeling with `moderndive`. Using these data science tools and helper functions from the `moderndive` package, you'll fit your first data models. In particular:
-    + Ch.\@ref(regression): Basic regression models with only one explanatory variable.
-    + Ch.\@ref(multiple-regression): Multiple regression models with more than one explanatory variable. 
+1. Data modeling with `moderndive`. Using these data science tools and helper functions from the `moderndive` package, you'll fit your first data models. In particular, you'll
+    + Ch.\@ref(regression): Discover basic regression models with only one explanatory variable.
+    + Ch.\@ref(multiple-regression): Examine multiple regression models with more than one explanatory variable. 
 1. Statistical inference with `infer`. Once again using your newly acquired data science tools, you'll unpack statistical inference using the `infer` package. In particular you'll:
     + Ch.\@ref(sampling): Learn about the role that sampling variability plays in statistical inference and the role that sample size plays in sampling variability.
-    + Ch.\@ref(confidence-intervals): Construct confidence intervals.
-    + Ch.\@ref(hypothesis-testing): Conduct hypothesis tests.
-1. Data modeling with `moderndive` (revisited): Armed with your understanding of statistical inference, you'll revisit and review the models you'll construct in Ch.\@ref(regression) & Ch.\@ref(multiple-regression). In particular you'll:
+    + Ch.\@ref(confidence-intervals): Construct confidence intervals using bootstrapping.
+    + Ch.\@ref(hypothesis-testing): Conduct hypothesis tests using permutation.
+1. Data modeling with `moderndive` (revisited): Armed with your understanding of statistical inference, you'll revisit and review the models you'll construct in Ch.\@ref(regression) and Ch.\@ref(multiple-regression). In particular, you'll:
```

Comment:
Made this match Ch11. Also testing out github.com GUI multi-line comment feature!

Reply:
I like it!